streaming/streaming.go

```diff
@@ -61,12 +58,15 @@ type DatastreamCfg struct {
 }
 
 type DataflowCfg struct {
-	JobName            string
-	Location           string
-	HostProjectId      string
-	Network            string
-	Subnetwork         string
-	DbNameToShardIdMap map[string]string
+	JobName             string
+	Location            string
+	HostProjectId       string
+	Network             string
+	Subnetwork          string
+	MaxWorkers          string
+	NumWorkers          string
```

Comment:
is this the minimum workers or default ?

Reply:
the initial number of workers